akka-stream/src/main/scala/akka/stream/javadsl/SourceWithContext.scala

```diff
@@ -38,7 +39,7 @@ object SourceWithContext {
  *
  * Can be created by calling [[Source.asSourceWithContext]]
  */
-final class SourceWithContext[+Out, +Ctx, +Mat](delegate: scaladsl.SourceWithContext[Out, Ctx, Mat])
+final class SourceWithContext[Out, Ctx, +Mat](delegate: scaladsl.SourceWithContext[Out, Ctx, Mat])
```

Comment:
Based on `javadsl.FlowWithContext` and `javadsl.Source` both being invariant in `Out` and `Ctx`, it looks like the former covariance was just an oversight.  In order to be able to use a `BiFunction` in `mapAsyncPartitioned`, the covariance was getting in the way.  Java of course will not respect the definition-site variances anyway.